docs/book/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "book",
+  "private": true,
+  "version": "1.0.0",
+  "description": "**Note:** Impatient readers head straight to [Quick Start](quick_start.md).",
+  "main": "index.js",
+  "dependencies": {
+    "cryptiles": "^4.1.2",
+    "gitbook-plugin-ga": "^1.0.1",
+    "gitbook-plugin-panel": "^0.0.1",
+    "gitbook-plugin-sequence-diagrams": "^1.1.0",
+    "gitbook-plugin-theme-api": "^1.1.2",
+    "phantomjs-prebuilt": "^2.1.16"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "sig-cluster-lifecycle",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kubernetes-sigs/cluster-api.git"
+  },
+  "keywords": [
+    "cluster-api"
+  ],
+  "bugs": {
+    "url": "https://github.com/kubernetes-sigs/cluster-api/issues"
+  },
+  "homepage": "https://github.com/kubernetes-sigs/cluster-api#readme"
```

Comment:
Should this be the netlify-based gitbook URL instead?

Reply:
Good catch. Until there is a consensus from #wg-k8s-infra, et al. on the use of Netlify, URL conventions, etc. this will be provisional but I think the URL should be `cluster-api.sigs.k8s.io` to match kind and kubebuilder. Let's leave this comment until I learn more.

Reply:
Can this link be updated now?

Reply:
Yes. I changed it to https://kubernetes-sigs-cluster-api.netlify.com since until https://github.com/kubernetes/k8s.io/pull/267 is merged the vanity URL will not exist...